feat: add row Count for upload section

khareyash05 opened the pull request:
Fixes #19 

Added row count for upload section
Working on adding the same for filter

ramonverse commented:
Hi @khareyash05 , thanks for putting this together. I refactored your code a bit and fixed a bug with the row count (should be just length, not length - 1).

khareyash05 commented:
> Hi @khareyash05 , thanks for putting this together. I refactored your code a bit and fixed a bug with the row count (should be just length, not length - 1).

Hey Thanks! I will be completing the filtering section now!
